pkg/keyidentifier/testdata/make_temp.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+
+# FIXME: this entire file should be deleted and merged with make.sh
+set -e
+
+function rand {
+    dd if=/dev/random bs=1 count=16 2> /dev/null  | base64
+}
+
+function makeUUID {
+    # od -x /dev/urandom | head -1 | awk '{OFS="-"; srand($6); sub(/./,"4",$5); sub(/./,substr("89ab",rand()*4,1),$6); print $2$3,$4,$5,$6,$7$8$9}'
+    ruby -r securerandom -e 'print SecureRandom.uuid'
+}
+
+# generate some keys and the corresponding json spec files for testing fingerprints
+function makeOpensshKeyAndSpec {
+    type=$1
+    bits=$2
+    format="openssh7"
+
+    keyfile=$(makeUUID)
+    keypath="fingerprints/$keyfile"
+
+    ssh-keygen -t $type -b $bits  -C "" -f $keypath -P /dev/null
+    ssh-keygen -t $type -b $bits  -C "" -f $keypath-encrypted -P "$(rand)"
+
+    for key in $keyfile $keyfile-encrypted; do
+        fingerprint=$(ssh-keygen -l -f fingerprints/$key | awk '{print $2}')
+        md5fingerprint=$(ssh-keygen -l -E md5 -f fingerprints/$key | awk '{print $2}' | sed 's/^MD5://')
+
+        encrypted=false
+        if [ "$key" == "$keyfile-encrypted" ]; then
+            encrypted=true
+        fi
+        echo "{ \"ExpectedFingerprintSHA256\": \"$fingerprint\",\
```

Comment:
I'd love to also get the command line that generated this in here.

(doesn't need to be in the go struct, just the json)